socorro/external/hb/crashstorage.py

```diff
@@ -10,7 +10,10 @@
 
```

Comment:
because, last I heard, we have external users that are still on HBase and not on S3